include/exadg/poisson/overset_grids/user_interface/application_base.h

```diff
@@ -90,6 +90,77 @@ locate_object_relative_to_triangulation(
     return Location::Intersected;
 }
 
+template<int dim>
+void
+set_boundary_ids_overlap_region(dealii::Triangulation<dim> const & tria_dst,
+                                dealii::types::boundary_id const & bid,
+                                dealii::Mapping<dim> const &       mapping_src,
+                                dealii::Triangulation<dim> const & tria_src)
+{
+  std::vector<dealii::Point<dim>> points;
+  using CellIteratorType = decltype(tria_dst.begin());
```

Comment:
```suggestion
  using CellIteratorType = dealii::Triangulation<dim>::typename cell_iterator;
```

Reply:
should probably read `typename dealii::Triangulation<dim>::cell_iterator`?